### DISCUSSION TITLE
Meaning of the `hyd_flag` parameter in pipeflow options

### DISCUSSION THREAD

**Discussion by amayboudellal** [e2nIEE/pandapipes#424] [Q&A]
Hi,

I'm trying to run a pipeflow on a district heating grid and specified `pp.pipeflow(net, mode='heat')` as it seemed to make the most sense. I then specified `pp.set_user_pf_options(net, reset=False)` but I get the error that I need to define `hyd_flag`. However, I cannot find any documentation or hint about what is meant by that and what are the expected arguments.

Could someone help me with that? Thanks.

**Comment by jkisse** [comment]
Hi @amayboudellal, if you choose `mode='heat'`, no hydraulic simulation will be conducted (only heat transfer). For hydraulic & heat transfer calculation, please choose `mode='all'`.
The `hyd_flag` indicates whether hydraulic results are available or not.
